cmake/simulation.cmake

```diff
@@ -51,5 +51,39 @@ macro(add_banshee_simulation name)
 	POST_BUILD
 	USES_TERMINAL
 	VERBATIM
-    )
+				)
+endmacro()
+
+# Function to create the flags for hyperflash
+# Saves the result into the `out_var`
+function(gvsoc_flags_add_files_to_hyperflash out_var files_var)
+	# LMACAN: The double variable expansion is needed because the first
+	# expansion gets us the variable name, the second one actual list elements
+	set(flags ${${files_var}})
+	list(TRANSFORM flags PREPEND "--flash-property=")
+	list(TRANSFORM flags APPEND "@hyperflash:readfs:files")
+	set(${out_var} ${flags} PARENT_SCOPE)
+endfunction()
+
+# The macro creates a new gvsoc_<name> cmake target which executes the final
+# binary on the gvsoc simulator. To give extra flags to the gvsoc command, set
+# the GVSOC_EXTRA_FLAGS variable.
+# The `v2` naming is temporary until we don't remove the equivalent command
+# used for the PULP and Siracusa targets.
```

Comment:
I do not get this comment.

Reply:
Thanks for pointing that out. It's an old comment that is not relevant anymore due to Victor's effort into synchronizing the gvsoc's :)

Reply:
Removed here: 62db05e512c6168f3469a03fe7d6a1ef6b5cf14d